code/modules/mob/living/carbon/carbon.dm

```diff
@@ -11,6 +11,15 @@
 	if(!mapload)  //I don't want no gas leaks on my space ruin you hear?
 		RegisterSignal(src, COMSIG_LIVING_DEATH, .proc/attach_rot)
 
+	//this is very slightly better than it was because you can use it more places. still can't do \his[src] though.
+	t_He = p_they(TRUE)
+	t_His = p_their(TRUE)
+	t_his = p_their()
+	t_him = p_them()
+	t_has = p_have()
+	t_is = p_are()
+	t_es = p_es()
+
```

Comment:
Couldn't this change during gameplay? At least in theory?

Reply:
Yeah, this gave me pause on this PR. Seems to be a very fragile cache and not worth it.

Reply:
Yeah I now realize why this is done this way (as cache) and just added the necessary vars to my proc instead of moving the cache